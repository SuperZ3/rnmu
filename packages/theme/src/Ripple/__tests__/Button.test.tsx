import { render } from '@testing-library/react-native';
import { Ripple } from '../index';
import { Text, View } from 'react-native';

describe('Ripple Component With Theme', () => {
  it('should match snapshot', () => {
    const ripple = render(
      <Ripple>
        <Text>ripple</Text>
      </Ripple>,
    ).toJSON();
    expect(ripple).toMatchSnapshot();
  });
});
