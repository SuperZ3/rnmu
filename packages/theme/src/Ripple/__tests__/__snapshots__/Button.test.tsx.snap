// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component With Theme should match snapshot 1`] = `"Hello World!"`;
